todo_app/data/trello_items.py

```diff
@@ -0,0 +1,58 @@
+from flask import session
+import requests
+import os
+import json
+
+trello_api_key = os.getenv('TRELLO_API_KEY')
+trello_api_token = os.getenv('TRELLO_API_TOKEN')
+trello_board_id = os.getenv('TRELLO_BOARD_ID')
+trello_to_do_list_id = os.getenv('TRELLO_TO_DO_LIST_ID')
+
+class Item:
+   def __init__(self, id, name, list):
+      self.id = id
+      self.name = name
+      self.list = list
+
+class List:

```

Comment:
You've taken a different (and arguably nicer!) approach than our exercise assumes here handling the lists more flexibly, which might mean the ViewModel in M3 feels very artificial and like it's moving backwards - so if you want to interpret it fairly differently that's fine, as long as you end up with a couple of sensible unit tests 😄